### ISSUE TITLE
RIDE 2.0a2 bugs

### ISSUE THREAD

**Issue by jaychhatrala** [robotframework/RIDE#2040]
Hello, 

I am using ride 2.0a2 version (got a link from github issues) as i was facing some problems with installation of older ride versions. Please check below my packages installed in the environment (made with python2.7 version).

I am using Ubuntu 16.04LTS version. 

```
(myenv) agile-33@agile-33:~/jc/tutorials/robot_framework$ pip list
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Package                        Version 
------------------------------ --------
docutils                       0.15.2  
numpy                          1.16.5  
Pillow                         6.2.1   
pip                            19.3.1  
pkg-resources                  0.0.0   
Pygments                       2.4.2   
PyPubSub                       3.3.0   
robotframework                 3.1.2   
robotframework-ride            2.0a2   
robotframework-seleniumlibrary 4.1.0   
robotframeworklexer            1.1     
selenium                       3.141.0 
setuptools                     41.4.0  
six                            1.12.0  
urllib3                        1.25.6  
wheel                          0.33.6  
wxPython                       4.0.0a2 
wxPython-common                2.8.12.1
```
Below is my main robot file:

```
*** Settings ***
Suite Setup       log to console    Test Suite started
Suite Teardown    log to console    Test Suite Completed
Library           SeleniumLibrary

*** Variables ***
${url}            https://opensource-demo.orangehrmlive.com/    # scalar url
@{CREDENTIALS}    Admin    admin123
&{LOGIN}          username=admin    password=admin123

*** Test Cases ***
TC1
    [Documentation]    My First Test Case in ride
    Open Browser    https://www.robotframework.org    gc
    close browser
    log to console    Opened Successfully

TC2
    [Documentation]    %{USER} is an environment variable. we can check this in terminal using echo $USER
    [Tags]    test2
    open browser    ${url}    gc
    input text    id=txtUsername    @{CREDENTIALS}[0]
    input password    id=txtPassword    &{login}[password]
    click button    id=btnLogin
    close browser
    log to console    %{USER} is logged in

TC3
    [Tags]    test3
    open Browser    ${url}    gc
    Login
    close browser

TC4
    [Tags]    test4
    [Setup]    open browser    ${url}    gc
    Login

*** Keywords ***
Login
    input text    id=txtUsername    Admin
    input password    id=txtPassword    admin123
    click button    id=btnLogin

GoToHomepage
    Login

```
All the tests are working correctly but i am having following issues with RIDE 2.0a2:

1)Cannot view the EDIT Pane many times, its having unusual behaviour, sometimes it displays the required information,  with some cases it works but with some cases it just does'nt show anything, its all blank and i cant do anything. please find below images for reference. 

![image](https://user-images.githubusercontent.com/51824329/68005520-8b608600-fc9b-11e9-8dc1-6616814e5910.png)

But not working here in below cases :
 
![image](https://user-images.githubusercontent.com/51824329/68005568-ac28db80-fc9b-11e9-984a-4744b11cc157.png)

![image](https://user-images.githubusercontent.com/51824329/68005582-b6e37080-fc9b-11e9-807f-ebb58650f9aa.png)

![image](https://user-images.githubusercontent.com/51824329/68005763-5274e100-fc9c-11e9-8fb5-76d33eef3b74.png)

2) In the Suite Edit pane, i have got some suite setup and suite teardown actions, it works when i am inside the edit button next to setup and teardown but once i click ok, it doesnt display in the main edit pane. Cant get you a screen shot right now as my main edit pane of the suite is not displaying anything. This even happens with the test case edit pane. 

3) SeleniumLibrary is installed and available in  python2.7>site-packages of the environment. Now when I am trying to import library using the edit pane of test suite in RIDE, its showing in RED color which means its not being imported but when i run the test cases selenium test cases are actually running. Please find below the screen shots and the test cases. Which means that seleniumlibrary is imported properly but its not displayed correctly that it has been imported correctly. Cannot show a screen shot of the library being displayed in red color as my edit pane is not displaying anything today. All my test case ran properly which means the selenium is working properly. 

![image](https://user-images.githubusercontent.com/51824329/68006040-52c1ac00-fc9d-11e9-9e9d-39bb33df038b.png)

Ride is a fantastic work to organise test cases and we are actively working with ride. Looking forward to get an apt solution. 

Many thanks in advance,

**Comment by HelioGuilherme66** [comment]
@jaychhatrala RIDE 2.0a2 is an old alpha release, that is not usable. It will be replaced with beta version, when development starts to remove python2.7 support. Current released version is 1.7.3.1, but since then, we have made a lot of fixes and improvements, so the newest released version is 1.7.4b2. You should install this last beta version, or even the current master development branch. Please navigate to [Releases](https://github.com/robotframework/RIDE/releases) and read the notes for the one to install. Take care that wxPython **should** be installed before RIDE (and there may be problems with old 2.8.12.1 version).

**Comment by jaychhatrala** [comment]
@HelioGuilherme66 Many thanks for suggesting me the beta version. I tried the 1.7.4b2 and it works perfect.
